index.css

```diff
@@ -0,0 +1,485 @@
+  word-break: break-all;
+  color: rgb(51, 51, 51);
+  border: 2px solid #fff;
+}
+
+.event-description {
+  background-color: rgb(229, 229, 229);
+}
+
+.location {
+  text-align: center;
+  border: none;
+  padding: 0;
+  margin: 0;
+  font-weight: bold;
+}
+
+.line {
+  width: 70px;
+  border-bottom: 1px solid rgb(239, 81, 60);
+}
+
+.speaker {
+  color: #222a32;
+  line-height: 24px;
+}
+
+.speaker-name {
+  font-size: 16px;
+}
+
+.speaker-job {
+  color: #ff4f38;
+  font-size: 13px;
+}
+
+.speaker-achievement {
+  font-size: 13px;
+}
+
+.full-program {
+  align-self: flex-end;
+  margin-bottom: 40px;
+}
+
+.full-link {
+  color: rgb(255, 255, 255);
+  font-size: 20px;
+  text-decoration: none;
+}
+
+.full-program:hover {
+  text-decoration: underline;
+}
+
+.greater {
+  width: 15px;
+}
+
+.partners {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: rgb(63, 63, 63);
+}
+
+.partner {
+  gap: 20px;
+}
+
+.partner-img {
+  width: 100%;
+  height: 100%;
+}
+
+.links,
+.social-media,
+.logo {
+  display: none;
+}
+
+.showlinks {
+  position: static;
+  display: flex;
+  flex-direction: row;
+}
+
+.copyright h4 {
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.copyright p {
+  font-size: 12px;
+  line-height: 24px;
+  margin-top: 12px;
+}
+
+.social-media a {
+  color: #fff;
+  font-size: 8px;
+  padding: 8px 0;
+}
+
+.links li,
+.logo {
+  padding: 7px 0;
+}
+
+.contact,
+.contact a {
+  font-size: 13px;
+  color: #222a32;
+  line-height: 24px;
+}
+
+.links a {
+  font-size: 18px;
+  color: #272a31;
+  font-weight: 600;
+}
+
+.links .event-border a {
+  border: 2px solid red;
+  padding: 5px 7px;
+  color: red;
+}
+
+.about-h {
+  font-size: 32px;
+  font-weight: bold;
+  font-family: Verdana, sans-serif;
+  text-transform: uppercase;
+  color: brown;
+}
+
+.activity {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+}
+
+.past-activies {
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.year-img img {
+  width: 100%;
+  opacity: 0.6;
+}
+
+.year {
+  width: 100%;
+  position: relative;
+}
+
+.past-activies h3 {
+  color: #39526b;
+  font-size: 20px;
+  line-height: 24px;
+  text-align: center;
+}
+
+.past-activies .line {
+  margin-bottom: 70px;
+}
+
+.event-year {
+  position: absolute;
+  top: 25%;
+  text-align: center;
+  left: 25%;
+}
+
+.event-year h4 {
+  color: #39526b;
+  font-size: 15px;
+  line-height: 24px;
+}
+
+@media screen and (max-width: 767px) {
+  .hamburger {
+    display: flex;
+    justify-content: space-between;
+    top: 6%;
+    position: fixed;
+    z-index: 2;
+  }
+
+  .hamburger img {
+    width: 40px;
+    cursor: pointer;
+  }
+
+  #hamburger {
+    margin-left: 30px;
+  }
+
+  #close {
+    width: 25px;
+    position: fixed;
+    right: 30px;
+  }
+
+  .showlinks {
+    flex-direction: column;
+    position: fixed;
+    padding-left: 6%;
+    padding-top: 20%;
+    background-color: rgb(32, 110, 189);
```

Comment:
- [ ] I kindly suggest that you make your hamburger menu background transparent or the same color as the top section background